### PR TITLE
Added lender and borrower accounts to seeds file

### PR DESCRIPTION
**Pull request by mikedao**
Added the lender and borrower accounts to the seeds file to aid in client demos.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -85,6 +85,8 @@
     t.datetime "created_at",   null: false
     t.datetime "updated_at",   null: false
     t.string   "slug"
+    t.boolean  "status"
```
Comment (oorion) [inline db/schema.rb]: do we need two columns for this?  Couldn't we have a status column that could be one of three things?  pending, approved, denied

File: db/seeds.rb [hunk 2]
```diff
@@ -59,6 +63,20 @@
 bridge_builders2 = Tenant.find(2)
 bridge_builders3 = Tenant.find(3)
 
+Tenant.find(1).users.create!(
+  username:               "borrower",
+  first_name:             "first",
+  last_name:              "last",
+  email:                  "borrower@borrower.com",
+  password:               "password",
```
Comment (Jwan622) [inline db/seeds.rb]: In my seed file, I created a user and then did user.tenant.create!(....) ... it's the same thing right? I created a tenant off a user whereas you created a user off of a tenant. There's no difference right? At the end of the day we have a borrower correct?

File: db/schema.rb (hunk 1 above)
Comment (Jwan622) [inline db/schema.rb]: Yeah I also am wondering why there needs to be status and approval boolean columns. A tenant is approved (true or false), but what is status for a tenant? Answer that and I can merge it in.

Comment (mikedao) [inline db/schema.rb]: Status is whether to decide if it is active or inactive.  Approval is if a platform admin has approved it or not. Each is a unique state, and i feel two columns gives us maximum flexibility.

Comment (EmilyMB) [inline db/schema.rb]: I feel OK about @mikedao, but whatever the team thinks is best :smile:

Comment (Jwan622) [inline db/schema.rb]: Awesome I agree.

**Comment by Jwan622** [comment]
Good stuff Mike.

**Pull request merged**